tools/tidy/tidy.cpp

```diff
@@ -0,0 +1,84 @@
+#include "fmt/color.h"
+#include "fmt/format.h"
+#include "include/TidyFactory.h"
+
+#include "slang/diagnostics/TextDiagnosticClient.h"
+#include "slang/driver/Driver.h"
+#include "slang/util/Version.h"
+
+using namespace slang;
+
+int main(int argc, char** argv) {
+
+    driver::Driver driver;
+    driver.addStandardArgs();
+
+    std::optional<bool> showHelp;
+    std::optional<bool> showVersion;
+    std::optional<bool> quiet;
+    driver.cmdLine.add("-h,--help", showHelp, "Display available options");
+    driver.cmdLine.add("--version", showVersion, "Display version information and exit");
+    driver.cmdLine.add("-q,--quiet", quiet, "Suppress non-essential output");
+
+    if (!driver.parseCommandLine(argc, argv))
+        return 1;
+
+    if (showHelp) {
+        slang::OS::print(
+            fmt::format("{}", driver.cmdLine.getHelpText("slang SystemVerilog linter")));
+        return 0;
+    }
+
+    if (showVersion) {
+        OS::print(
+            fmt::format("slang-tidy version {}.{}.{}+{}\nCompiled with slang version {}.{}.{}+{}\n",
```

Comment:
Probably easiest to just use the slang version for slang-tidy as well. Hardcoding the version number here isn't that useful.